src/components/UI/Text/Textarea.vue

```diff
@@ -114,6 +118,12 @@ export default {
       type: Number,
     },
   },
+  data() {
+    return {
+      editor: null,
```

Comment:
this is an instance of the quilljs editor right?
maybe rename it to `quillInstance` or `quillEditor`?

Reply:
Also add comments!

Reply:
Instead of textarea or inputarea, I'll rename it as quillEditor then